src/Pipedrive.net/Clients/LeadsClient.cs

```diff
@@ -39,5 +40,12 @@ public Task<Lead> Get(Guid id)
         {
             return ApiConnection.Get<Lead>(ApiUrls.Lead(id));
         }
+
+        public Task<LeadCreated> Create(NewLead data)
+        {
+            Ensure.ArgumentNotNull(data, nameof(data));
+
+            return ApiConnection.Post<LeadCreated>(ApiUrls.Leads(), JObject.FromObject(data));
```

Comment:
Using `JObject.FromObject` construction to handle null values.
Without null values handling receive error like:
```json
{
   "success":false,
   "data":null,
   "additional_data":null,
   "error":"provided dataset is not valid",
   "error_info":"\"owner_id\" must be a number"
}
```

Example of request:

```csharp
var data = new NewLead("Lead created from C#")
{
    OrganizationId = 1
};
var createdLead = await client.Lead.Create(data);
```